### ISSUE TITLE
Download stops itself

### ISSUE THREAD

**Issue by DheekeyCarey** [JunkFood02/Seal#1637]
### Checklist

- [X] I'm reporting a bug unrelated to a specific site.
- [X] I've verified that I'm running the [**latest version**](https://github.com/yt-dlp/yt-dlp/releases/latest) of yt-dlp.
- [X] I've verified that I'm running the latest [**stable version**](https://github.com/JunkFood02/Seal/releases/latest/) of Seal or any later [**preview versions**](https://github.com/JunkFood02/Seal/releases).
- [X] I've read the [**Contributing guidelines**](https://github.com/JunkFood02/Seal/blob/main/CONTRIBUTING.md) and [**Code Of Conduct.**](https://github.com/JunkFood02/Seal/blob/main/CODE_OF_CONDUCT.md)
- [X] I've checked that the site i'm trying to download from is in the [**Supported Sites**](https://github.com/yt-dlp/yt-dlp/blob/master/supportedsites.md) list from yt-dlp
- [X] I understand that the issue will be (ignored/closed) if I intentionally remove or skip any mandatory field.

### Describe the bug

The download(s) will always stop not long after I close the notification tray. The download won't stop if I keep the tray open or shown. So I have to wait for the download to finish before I can proceed doing other things with my phone.

### Error reports

No reports. It just terminates itself.

### Screenshots & Screen Records

https://github.com/user-attachments/assets/350d8455-81c3-4a32-b18f-441b07987d09

### Additional context

It happens in both stable and preview versions. Regardless the main app in the background is open 
and running or not.

**Comment by JunkFood02** [comment]
Did you allow Seal to running in background in the battery setting?

**Comment by DheekeyCarey** [comment]
> Did you allow Seal to running in background in the battery setting?

Yes, that was the first thing came to my mind. but unfortunately
![Screenshot_2024-08-16-19-34-26-934_com android settings](https://github.com/user-attachments/assets/3aef0a76-e48a-4e41-b1e9-9ef03bfb3a45)
 it doesn't help at all

**Comment by schaurian** [comment]
I tried it with yt-dlp 2024.08.06. It worked perfectly. So i guess seal (or Samsung) is the issue.
